boards/nordic/nrf52840dk/src/main.rs

```diff
@@ -107,7 +112,7 @@ const FAULT_RESPONSE: kernel::procs::FaultResponse = kernel::procs::FaultRespons
 const NUM_PROCS: usize = 8;
 
 #[link_section = ".app_memory"]
-static mut APP_MEMORY: [u8; 245760] = [0; 245760];
+static mut APP_MEMORY: [u8; 0x3A000] = [0; 0x3A000];
```

Comment:
This is probably a good change, but a weird thing to have in this series

Reply:
The app memory needed to be decreased because the Segger RTT takes more space off kernel memory. I figured it would be clearer to keep this size in hexadecimal as well.

Reply:
I checked again and could increase the app memory size again.